tests/defaultAst/DbaInferenceTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace staabm\PHPStanDba\Tests;
+
+use PHPStan\Testing\TypeInferenceTestCase;
+use function getenv;
+
+class DbaInferenceTest extends TypeInferenceTestCase
```

Comment:
I think we should rename the test-suite fro `defaultAst` to `sqlAst` and the test from `DbaInferenceTest` to `SqlAstInferenceTest`

Reply:
done